pytest_html/plugin.py

```diff
@@ -95,23 +98,27 @@ def __init__(self, logfile):
         self.passed = self.skipped = 0
         self.xfailed = self.xpassed = 0
         self.rerun = 0
+        self.self_contained = self_contained
 
     class TestResult:
 
-        def __init__(self, outcome, report):
+        def __init__(self, outcome, report, self_contained,
+                     logfile, test_index):
```

Comment:
Since the reruns starts on 0, I'll let the default as -1, so we can know which was the first rerun and the failed test